packages/react-sdk/package.json

```diff
@@ -5,6 +5,7 @@
   "author": "Webstudio <github@webstudio.is>",
   "homepage": "https://webstudio.is",
   "scripts": {
+    "dev": "tsc -w",
```

Comment:
using tsc for compiling is slow, that's why I suggested using tsup, because it uses esbuild, which is just stripping off the types

Reply:
Ah, I see.  Let me move it to `tsup` then.

Reply:
imagine you have 20 packages and start yarn dev in the root, so you start watching 20 packages and compiling them all with typescript ... its a nightmare

Reply:
Yeah, right!